### PR TITLE
Fe add comment molecule

### PR DESCRIPTION
**Pull request by will0603**
I implemented the Comment molecule component, it consists of the reuse of the iconCircle atom component, adding the number of hearts, comments, shares, statistics, and stars, in addition the data of the user who makes the comment was added as the user to whom it is sent the comment.
In this commit the iconCircle atom component was also modified, I had to add a new type of color called "comment", with this type I was able to add the necessary styles that are according to what was designed in the figma (Frame 241).

https://www.figma.com/file/WyhCIiSidyM6aXsbxdu1rI/Dise%C3%B1o-UX%2FUI--App-Social?type=design&node-id=1117-3115&mode=design&t=qqijLjPa2g5I6cPZ-0

### CODE REVIEW

File: frontend/src/components/molecules/Comment/comment.tsx [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  comment?: string;
+  fullWidth?: boolean;
+  countHeart?: number;
+  countComment?: number;
+  countShareds?: number;
+  countStatistics?: number;
+  countStars?: number;
+}
+
+const WComment: React.FC<WCommentProps> = ({
```
Comment (AnthonyRicse98) [inline frontend/src/components/molecules/Comment/comment.tsx:28]: make a more organized code, structured by libraries, interface, component coding, defaults props

reference  : https://github.com/unmsmfisi-socialapplication/social_app/pull/318/files  

use file scss  ,your file is growing too much, and therefore it is better to separate it into a scss file


image

**Review by AnthonyRicse98 — COMMENTED** [review]
repair

File: frontend/src/components/molecules/Comment/comment.tsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    name: "Milly"
+  },
+  sendUser: "janedoel123",
+  comment: "Lorem ipsum dolor sit amet consectetur. Congue et nunc sed nascetur malesuada",
+  countHeart: 12,
+  countComment: 5,
+  countShareds: 6,
+  countStatistics: 126,
+  countStars: 23
+}
```
Comment (Anthonyricse) [inline frontend/src/components/molecules/Comment/comment.tsx:123]: your  component is not working 
image
fix the css that must be failing

File: frontend/src/components/molecules/Comment/index.scss [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    gap: 5px
+}
+
+span{
+    color: #878787
+}
+
+strong{
+    color: #000
+}
```
Comment (Anthonyricse) [inline frontend/src/components/molecules/Comment/index.scss:39]: Reference BEM ; https://getbem.com/
html tags used span , strong  , a  , should be inside the classname of the div respectively.
use nomenclature BEM 
image
EXAMPLE
.opinions_box {
    margin: 0 0 8px 0;
    text-align: center;

    &__view-more {
        text-decoration: underline;
    }

    &__text-input {
        border: 1px solid #ccc;
    }

    &--is-inactive {
        color: gray;
    }
}

**Review by AnthonyRicse98 — APPROVED** [review]

**Pull request merged**